install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -14,6 +14,14 @@ ALTER TABLE `PREFIX_product_attribute` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 
 /* Delete price display precision configuration */
+
+DELETE `PREFIX_configuration` WHERE name = 'PS_PRICE_DISPLAY_PRECISION';
```

Comment:
this was added by a merge, but please note that **delete statement is missing FROM**

Reply:
You need to use rebase command instead of `git pull` / `git merge`

Reply:
fixed it